nengo_gui/guibackend.py

```diff
@@ -102,17 +103,47 @@ class GuiRequestHandler(server.HttpWsRequestHandler):
         '/favicon.ico': 'serve_favicon',
     }
 
-    def get_expected_origins(self):
-        session = self.get_session()
+    def _is_loopback(self, host):
```

Comment:
It would say something like "Checks whether `host` refers to the loopback interface." Not sure if that really adds any value because to me it is what the function name already says in more words.